robot-user/setup

```diff
@@ -16,3 +16,18 @@ function hostname-core
 alias topic-monitor='rosrun robot_launch_files topic_monitor */scan */rgbd */joint_states */base/measurements'
 
 alias scores='scp amigo@athome10.local:/home/amigo/database/per_challenge/scores.html /tmp/scores.html && google-chrome /tmp/scores.html'
+
+cleanup()
+{
+  # Some cache clearing to free diskspace. In order of quickness
+  echo "Cleaning ccache"
+  ccache --clean
+  echo "Cleaning apt-get cache"
+  sudo apt-get clean
+  echo "Cleaning ros logs"
+  rosclean purge -y
+  echo "Cleaning docker files"
+  docker system prune
+  echo "Cleaning old kernels"
+  echo $(dpkg --list | grep linux-image | awk '{ print $2 }' | sort -V | sed -n '/'`uname -r`'/q;p') $(dpkg --list | grep linux-headers | awk '{ print $2 }' | sort -V | sed -n '/'"$(uname -r | sed "s/\([0-9.-]*\)-\([^0-9]\+\)/\1/")"'/q;p') | xargs sudo apt-get -y purge
+}
```

Comment:
I am not sure if we want to provide this. @ar13pit, what do you think?

Reply:
Me neither. Don't care about this one, just thought I might be useful.

Feel free to try the function on your machine and see how much you gain :slightly_smiling_face: